### PR TITLE
keycat support bugs fixes

### PR DESCRIPTION
**Pull request by s-kashko**

### CODE REVIEW

File: app/containers/SendTips/CurrencyField.jsx [hunk 1]
```diff
@@ -124,19 +126,19 @@ const CurrencyField = ({
           <Label>
             {translationMessages[locale]?.[messages.sendFromAccount.id]}
           </Label>
-          {!isPeer && (
-            <B onClick={selectAccount} type="button">
-              {(isScatterWalletSelected || isKeycatWalletSelected) && (
+          {!isPeer &&
+            !(withScatter && isScatterWalletSelected) &&
```
Comment (sergeyi23) [inline app/containers/SendTips/CurrencyField.jsx:130]: enums would definitely help simplify this logic. This comparison would be as simple as ` currentWallet == selectedWallet`

File: app/containers/SendTips/SendTipsForm.jsx [hunk 2]
```diff
@@ -324,10 +328,27 @@ FormClone = connect(
...
+      const isPreselectedInWalletsArray =
+        tipsPreselect &&
+        wallets.find(wallet => wallet.name === tipsPreselect[WALLET_FIELD]);
+
+      if (isPreselectedInWalletsArray)
+        return Object.values(WALLETS).find(
+          wallet => wallet.name === tipsPreselect[WALLET_FIELD],
+        );
+
+      // there is no saved data for send tips
```
Comment (sergeyi23) [inline app/containers/SendTips/SendTipsForm.jsx:341]: wallets[0], wallets[1], wallets[2] are magic values that require guessing what these indexes mean.

**Review by sergeyi23 — APPROVED** [review]
There are few refactoring suggestions. You should try using enum for wallet related variables.

File: app/containers/SendTips/SendTipsForm.jsx (hunk 2 above)
Comment (s-kashko) [reply app/containers/SendTips/SendTipsForm.jsx:341]: fixed

**Pull request merged**